schema_based_on_diagram.sql

```diff
@@ -0,0 +1,77 @@
+CREATE DATABASE schema;
+\c schema;
+-- Create table named PATIENTS
+CREATE TABLE patients(
+    id int primary key,
+    name varchar(255),
+    date_of_birth date
+);
+
+-- Create table named medical histories
+CREATE TABLE medical_histories(
+    id int primary key,
+    admitted_at timestamp,
+    patient_id int,
+    status varchar(255)
+);
```

Comment:
-  Kindly reference the patients table using the patients_id as a foreign key in medical_histories.
For example:
```
FOREIGN KEY (patients_id) REFERENCES patients(id)
```